src/js/remoteServices.js

```diff
@@ -184,4 +184,14 @@ export default {
     },
     allowHostOverride: false,
   },
+
+  basecamp3: {
+    name: "basecamp3",
+    host: "https://3.basecamp.com",
+    urlPatterns: [":host:/:instanceId/buckets/:projectId/todos/:todoId"],
+    projectId: (document, service, { projectId }) => projectId,
+    taskId: (document, service, { todoId }) => todoId,
```

Comment:
So if I rename `todoId` to `id`, this line:
`taskId: (document, service, { id }) => id`
becomes redundant, because this is the default behavior anyways. Did I understand that correctly?